public/locale/en.json

```diff
@@ -48,6 +48,7 @@
   "DOMESTIC_HEALTHCARE_SUPPORT__NO_SUPPORT": "No support",
   "DOMESTIC_HEALTHCARE_SUPPORT__PAID_CAREGIVER": "Paid caregiver",
   "ENCOUNTER_TAB__abdm": "ABDM Records",
+  "ENCOUNTER_TAB__claims": "Insurance Claims",
```

Comment:
shouldn't this be present in the plug instead?

Reply:
No, this can't be done in the plug right now, we need some extra config to do that, will take that up in a separate PR.